tutorials/parameter_assignment_assembler_setup.ipynb

```diff
@@ -264,65 +284,144 @@
     "# needed for a multi-physics problem.\n",
     "\n",
     "# Define a grid\n",
-    "gb = make_grid()\n",
+    "gb = make_grid(N=[4, 4])\n",
     "parameter_keyword = 'second_flow_param'\n",
     "gb = assign_data(gb, parameter_keyword, parameter_keyword_2)\n",
     "\n",
     "\n",
     "# Primary variables must be defined on each component of the GridBucket.\n",
     "# Define the pressure variable with the same keyword on all grids\n",
-    "grid_1_variable_1 = 'pressure'\n",
-    "grid_1_variable_2 = 'foo'\n",
+    "grid_1_pressure_variable = 'pressure'\n",
+    "grid_1_temperature_variable = 'temperature'\n",
+    "\n",
+    "# On the second grid, we use a mixed method with both pressure and fluxes as primary variables\n",
+    "grid_2_pressure_variable = 'flux_pressure'\n",
+    "# Temperature also on the second grid\n",
+    "grid_2_temperature_variable = 'temperature'\n",
+    "\n",
+    "# Coupling variable for pressure\n",
+    "mortar_variable_pressure = 'mortar_flux_pressure'\n",
+    "# Coupling variable for advective temperature flux\n",
+    "mortar_variable_temperature_1 = 'mortar_flux_diffusion'\n",
+    "mortar_variable_temperature_2 = 'mortar_flux_diffusion_2'\n",
+    "\n",
+    "# Identifier of the discretization operator for pressure discretizaiton\n",
+    "operator_keyword_pressure = 'pressure_diffusion'\n",
+    "\n",
+    "# identifier for the temperature discretizations.\n",
+    "# THIS IS WEIRD: The intention is to illustrate the use of two discretization operators for \n",
+    "# a single variable. The natural option in this setting is advection-diffusion, but that\n",
+    "# requires either the existence of a Darcy flux, or tighter coupling with the pressure equation.\n",
+    "# Purely for illustrative purposes, we instead use a double diffusion model. There you go.\n",
+    "operator_keyword_temperature_1 = 'diffusion'\n",
+    "operator_keyword_temperature_2 = 'diffusion_2'\n",
```

Comment:
I think part of the problem might be that you try to illustrate (almost) all possible types of combinations in a single example. The example is quite extensive and not altogether straightforward to grasp, perhaps it could be an option to have a different example illustrating the use of two discretization operators for a single variable (e.g. advection-diffusion with fixed Darcy flux).